hazelcast-client/src/test/java/com/hazelcast/client/starter/HazelcastClientStarter.java

```diff
@@ -39,36 +39,36 @@ public static HazelcastInstance newHazelcastClient(String version, boolean enter
         return newHazelcastClient(version, null, enterprise);
     }
 
+    @SuppressWarnings("unchecked")
     public static HazelcastInstance newHazelcastClient(String version, ClientConfig clientConfig, boolean enterprise) {
         ClassLoader classLoader = clientConfig == null ? null : clientConfig.getClassLoader();
         HazelcastAPIDelegatingClassloader classloader = getTargetVersionClassloader(version, enterprise, classLoader);
         ClassLoader contextClassLoader = currentThread().getContextClassLoader();
         currentThread().setContextClassLoader(null);
         try {
-            Class<Hazelcast> hazelcastClass = (Class<Hazelcast>) classloader.loadClass("com.hazelcast.client.HazelcastClient");
+            Class<HazelcastClient> hazelcastClass
```

Comment:
Is there any reason that `HazelcastClient` was not used before? This fix helps the reflection support of IDEA, which can now find `newHazelcastClient` in line 55.